### PR TITLE
More general language used

### PR DESCRIPTION
**Pull request by Futochu**
Replaced instances of (F|f)uckup with (M|m)istake

### CODE REVIEW

File: imports/ui/HongKongNewGame.js [hunk 1]
```diff
@@ -723,12 +723,12 @@ function push_mistake_hand(template) {
...
-	else if (loserWind == "south") Session.set("southFuckupTotal", Number(Session.get("southFuckupTotal")) + 1);
-	else if (loserWind == "west")  Session.set("westFuckupTotal",  Number(Session.get("westFuckupTotal"))  + 1);
-	else if (loserWind == "north") Session.set("northFuckupTotal", Number(Session.get("northFuckupTotal")) + 1);
+	if 		(loserWind == "east")  Session.set("eastMistakeTotal",  Number(Session.get("eastMistakeTotal"))  + 1);
+	else if (loserWind == "south") Session.set("southMistakeTotal", Number(Session.get("southMistakeTotal")) + 1);
+	else if (loserWind == "west")  Session.set("westMistakeTotal",  Number(Session.get("westMistakeTotal"))  + 1);
+	else if (loserWind == "north") Session.set("northMistakeTotal", Number(Session.get("northMistakeTotal")) + 1);
 
-	pushHand(template, "fuckup", eastDelta, southDelta, westDelta, northDelta);
+	pushHand(template, "mistake", eastDelta, southDelta, westDelta, northDelta);
```
Comment (TwelveNights) [inline imports/ui/HongKongNewGame.js]: Could you replace this with the constant?

File: imports/ui/JapaneseNewGame.js [hunk 2]
```diff
@@ -1044,12 +1044,12 @@ function push_mistake_hand(template) {
...
-	else if (loserWind == "south") Session.set("southFuckupTotal", Number(Session.get("southFuckupTotal")) + 1);
-	else if (loserWind == "west")  Session.set("westFuckupTotal",  Number(Session.get("westFuckupTotal"))  + 1);
-	else if (loserWind == "north") Session.set("northFuckupTotal", Number(Session.get("northFuckupTotal")) + 1);
+	if 		(loserWind == "east")  Session.set("eastMistakeTotal",  Number(Session.get("eastMistakeTotal"))  + 1);
+	else if (loserWind == "south") Session.set("southMistakeTotal", Number(Session.get("southMistakeTotal")) + 1);
+	else if (loserWind == "west")  Session.set("westMistakeTotal",  Number(Session.get("westMistakeTotal"))  + 1);
+	else if (loserWind == "north") Session.set("northMistakeTotal", Number(Session.get("northMistakeTotal")) + 1);
 
-	pushHand(template, "fuckup", eastDelta, southDelta, westDelta, northDelta);
+	pushHand(template, "mistake", eastDelta, southDelta, westDelta, northDelta);
```
Comment (TwelveNights) [inline imports/ui/JapaneseNewGame.js]: Same grievance as before

File: imports/ui/HongKongNewGame.js (hunk 1 above)
Comment (Futochu) [inline imports/ui/HongKongNewGame.js]: I was considering that, but I wasn't sure if it should be the same constant as the one used for the HTML tag display.  It seems sloppy to mix those two constants.  Opinions?

Comment (TwelveNights) [inline imports/ui/HongKongNewGame.js]: Mmm, I think it's still okay in terms of using the mistake on line 46 of `Constants.js`. The value will (hopefully) not be changing any time soon, and if it does, they change together. Having it as that constant may be more appropriate.

Comment (Futochu) [inline imports/ui/HongKongNewGame.js]: This is my new strategy to get work done.  This commit is strictly for cleaning up language, not ensuring constant usage.  I think this is good as it is.

**Pull request merged**